manifests/check.pp

```diff
@@ -49,6 +49,7 @@
   $package_name = 'monit',
   $service_name = 'monit',
 ) {
+  include monit::params
```

Comment:
This line is compliant with the best practices formulated by Puppet Labs : https://docs.puppetlabs.com/puppet/latest/reference/lang_defined_types.html#defining-a-type.
So I am happy with this.